src/tools/fuzzing/fuzzing.cpp

```diff
@@ -851,12 +851,16 @@ void TranslateToFuzzReader::addImportCallingSupport() {
 
   if (choice & 1) {
     // Given an export index, call it from JS.
+    // A second parameter has flags. The first bit determines whether we catch
+    // and rethrow all exceptions. (This ends up giving us the same signature
+    // and behavior as the normal mode, so we just add the flags here rather
```

Comment:
What's "normal mode"?

Reply:
"normal" is when the bit is not set. I'll clarify the comment.